WebDriverAgentLib/Utilities/FBConfiguration.m

```diff
@@ -139,6 +143,21 @@ + (void)setShouldWaitForQuiescence:(BOOL)value
 
 #pragma mark Private
 
++ (NSUInteger)mjpegServerPortFromArguments
+{
+  NSArray *arguments = NSProcessInfo.processInfo.arguments;
+  NSUInteger index = [arguments indexOfObject:@"--mjpeg-server-port"];
+  if (index == NSNotFound || index == arguments.count - 1) {
+    return NSNotFound;
+  }
+  NSString *portNumberString = arguments[index + 1];
+  NSUInteger port = (NSUInteger)[portNumberString integerValue];
+  if (port == 0) {
```

Comment:
I'd rather use a ternary operator here

Reply:
I decided to keep code similar to `bindingPortRangeFromArguments` as much as possible for consistency.

Reply:
Then there is an unnecessary code duplication. Please extract the duplicated code into a separate helper method

Reply:
It is different, as the binding port usage logic was designed to use a range of ports, and mjpeg server port option was design to be used as a single value.

Reply:
The code where the value is parsed from arguments list is exactly the same